kani-dependencies

```diff
@@ -1,4 +1,7 @@
 CBMC_VERSION="5.79.0"
-# If you update this version number, remember to bump it in `src/setup.rs` too
-CBMC_VIEWER_VERSION="3.8"
+CBMC_VIEWER_VERSION=$(
```

Comment:
We wanted to use `kani-dependencies` as the single source of truth for versions of dependencies, so I would rather keep the version explicit here and instead update `src/setup.rs` to read the version from this file.

Reply:
Thanks for catching that.

Reply:
@zhassan-aws as part of @jaisnan's release script work, we will also need to store the GitHub repositories of each of these tools somewhere. It would be nice not to have to store them in two different places.

Can I suggest that we convert this file into JSON? The file is only ever used from the various `install_*.sh` scripts that run in CI. Thus, we could continue to read the versions if the CI has `jq` available. My proposal is to write the file like this:

```json
{
  "CBMC_VIEWER": {
    "version": "3.8",
    "url": "https://github.com/model-checking/cbmc-viewer",
  }
}
```

Then `install_cbmc.sh` would do
```sh
CBMC_VERSION=$(jq -e CBMC.version < kani-dependencies.json)
```
and `setup.rs` would also parse the file with a JSON parser to find the CBMC Viewer version.

What do you think?

Reply:
Converting it to a JSON file is definitely a good idea.

Reply:
Actually, I don't understand why we want JSON here.